etc/kayobe/stackhpc-monitoring.yml

```diff
@@ -15,3 +15,13 @@ alertmanager_low_memory_threshold_gib: 5
 # Enabling this flag will result in HAProxy configuration and Prometheus scrape
 # targets being templated during deployment.
 stackhpc_enable_os_capacity: false
+
+# Keystone authentication URL for OpenStack Capacity
+stackhpc_os_capacity_auth_url: "{{ public_openrc_auth_url }}"
```

Comment:
`public_openrc_auth_url` is only defined in the context of the `ansible/public-openrc.yml` playbook in kayobe, and would not be valid here.